mlir-tensorrt/compiler/lib/Dialect/Plan/Transforms/Passes.cpp

```diff
@@ -73,6 +76,11 @@ struct ClusteringPipelineCliOpts
     : public PassPipelineOptions<ClusteringPipelineCliOpts> {
   Option<std::string> entrypoint{*this, "entrypoint", llvm::cl::init(""),
                                  llvm::cl::desc("name of entrypoint function")};
+  Option<bool> useNonDPSCallConv{
+      *this, "use-non-dps-call-conv",
+      llvm::cl::desc(
+          "allow tensorrt based output allocations using output allocator"),
```

Comment:
Please correct doc comment throughout similar to other comments for this option.